biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -36,33 +36,27 @@ label {
   text-align: left;
 }
 
-.largeBatteryCells {
-  display: inline-block;
-}
-
-.smallBatteryCells {
-  display: inline-block;
-  margin-right: 1.25rem;
-  margin-bottom: 1.25rem;
-}
-
-.solarPanels {
-  display: inline-block;
+.submitButton {
+  text-align: center;
 }
 
-.startDatePicker {
+.tooltip {
+  position: relative;
   display: inline-block;
-  margin-top: 1.25rem;
-  margin-right: 1.25rem;
-  margin-bottom: 1.25rem;
 }
 
-.submitButton {
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
```

Comment:
ni: if we can keep all colors in hex code that would be great rather than having some rgb and some hex